editor.planx.uk/src/ui/editor/RichTextInput.tsx

```diff
@@ -96,15 +96,6 @@ export const RichContentContainer = styled(Box)(({ theme }) => ({
         fontWeight: "inherit",
       },
     },
-    // Styles for placeholder text, to match ui/Input.tsx
-    "& p.is-editor-empty:nth-child(1)::before": {
-      color: theme.palette.text.placeholder,
-      opacity: 1,
-      content: `attr(data-placeholder)`,
-      float: "left",
-      height: 0,
-      pointerEvents: "none",
-    },
```

Comment:
Removed as there's no reference to the class `is-editor-empty` elsewhere in the codebase, I believe this is redundant.

Reply:
@DafyddLlyr I see that there's no longer placeholder text on the Editor Model:

<img width="1296" alt="Screenshot 2024-10-07 at 11 05 19" src="https://github.com/user-attachments/assets/626f5656-c2fd-4375-89cc-de7a9b28eaa5">

Wonder if this is linked to this change being removed?

Reply:
Aha - I missed this somehow - taking a look thanks!

Reply:
Fixed please see 3af7314735349a77a817510f4d41a79653faa482

This was actually a class generated by tiptap ([docs](https://tiptap.dev/docs/editor/extensions/functionality/placeholder#additional-setup)).